auth-web/src/util/common-util.ts

```diff
@@ -12,6 +12,11 @@ export default class CommonUtils {
     return value?.startsWith('http')
   }
 
+  // Format payment account display name
```

Comment:
no need for the comment? could rename the function to exactly this if needed

Reply:
removed comment